Partition (WBP)/Functions/VI0034C.VTI0034.fun.yml

```diff
@@ -36,11 +35,16 @@
     Clr_List Named(#wLRet)
     #Com_Owner.mthAdd_Msg( OK *Function ('Start - ' + *Function_Desc) )
     
+    #Com_Owner.mthGetExecutionType()
+    
     Get_Entry Number(1) From_List(#wLKData)
```

Comment:
Delete this line. Its included in mthGetExecutionType()

Reply:
this duplicate line has been removed